src/Microsoft.SqlTools.ServiceLayer/HostLoader.cs

```diff
@@ -94,6 +95,9 @@ private static void InitializeRequestHandlersAndServices(ServiceHost serviceHost
             DisasterRecoveryService.Instance.InitializeService(serviceHost);
             serviceProvider.RegisterSingleService(DisasterRecoveryService.Instance);
 
+            SerializationService.Instance.InitializeService(serviceHost);
```

Comment:
Why are you initializing this here? Isn't this the one place you don't want the serialization service? I thought you'd go through the existing path for requests to the SqlToolsService since it doesn't have to pipe data out and back into itself?

Reply:
If we don't need to initialize the serialization service here, then I'll go ahead and remove the initialization.